### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by erikwiffin**

### CODE REVIEW

File: tests/models/test_participants.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    try:
+        # pylint: disable=protected-access
+        participant._authorizations[0].status = participants.Authorization.STATUS_ACTIVE
+    finally:
+        pass
+    assert participant.authorization(provider)
+
+    # Once they've been activated, they can't be re-authorized
+    with pytest.raises(participants.AuthorizationException):
+        participant.authorize(provider, 'NEW_AUTHORIZATION_CODE', 'STATE')
```
Comment (jmandel) [inline tests/models/test_participants.py]: This is a surprising restriction - after a year, or after a participant manually revokes access, we'd want the ability to reauthorize. Is there a different workflow for that use case?

Comment (erikwiffin) [inline tests/models/test_participants.py]: It's only a restriction for "active" authorizations. If an authorization expires, or is revoked, you'd be able to reauthorize. (not that this system currently has a mechanism for expiring or revoking authorizations)

Comment (jmandel) [inline tests/models/test_participants.py]: Haha,  ok.  Maybe we can allow reauthorization of active instances  until we have working support for de-activation?

Comment (erikwiffin) [inline tests/models/test_participants.py]: Ended up being pretty easy to handle de-activation.

**Pull request merged**